cmd/pebble/main.go

```diff
@@ -33,6 +35,10 @@ func main() {
 		"strict",
 		false,
 		"Enable strict mode to test upcoming API breaking changes")
+	resolverAddress := flag.String(
+		"dnsserver",
+		"",
+		"Define a custom DNS server address (ex: 192.168.0.56:5053 or 8.8.8.8:53). Override the configuration file.")
```

Comment:
I think the "Override the configuration file." part can be removed now :scissors: I edited the PR description to remove the configuration file reference already.